providers/instantly/parse.go

```diff
@@ -0,0 +1,59 @@
+package instantly
+
+import (
+	"strconv"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/jsonquery"
+	"github.com/amp-labs/connectors/common/urlbuilder"
+	"github.com/spyzhov/ajson"
+)
+
+func makeGetTotalSize(dataLocation string) common.ListSizeFunc {
+	return func(node *ajson.Node) (int64, error) {
+		return jsonquery.New(node).ArraySize(dataLocation)
+	}
+}
+
+func makeGetRecords(dataLocation string) common.RecordsFunc {
```

Comment:
This and above function should be in common pacakge or utils package and imported to be used. Also `return jsonquery.New(node).ArraySize(dataLocation)` is being used twice for no reason and drawing performance back. 

Please resolve these before merging

Reply:
I initially approved, but found there are some code `makeGetRecords` that is redefined for each connector and the function has nothing specific to the connector. Please move to common or utils and import to use.

Reply:
I will create common utilities for this method.

Reply:
Dependent PR: https://github.com/amp-labs/connectors/pull/1004